src/libraries/System.Private.Runtime.InteropServices.JavaScript/System.Private.Runtime.InteropServices.JavaScript.sln

```diff
@@ -15,6 +15,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "src", "src", "{F323228E-200
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "gen", "gen", "{53D4886A-6035-458C-834C-D7EDC02A0E26}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Net.Http", "..\System.Net.Http\src\System.Net.Http.csproj", "{BB41B80A-455B-40D2-98A0-6663DDB4C1FD}"
```

Comment:
Why did the sln change so much?

Reply:
I added all the projects which depend on private interop